pkg/adapter/nimbus-kyverno/manager/manager.go

```diff
@@ -177,12 +175,15 @@ func createOrUpdateKp(ctx context.Context, npName, npNamespace string) {
 				logger.Info("KyvernoPolicy created", "KyvernoPolicy.Name", kp.Name, "KyvernoPolicy.Namespace", kp.Namespace)
 			}
 		} else {
-			kp.ObjectMeta.ResourceVersion = existingKp.ObjectMeta.ResourceVersion
-			if err = k8sClient.Update(ctx, &kp); err != nil {
-				logger.Error(err, "failed to configure existing KyvernoPolicy", "KyvernoPolicy.Name", existingKp.Name, "KyvernoPolicy.Namespace", existingKp.Namespace)
-				return
+			reason, isEqual := utils.PolEqual(existingKp, kp)
+			if !isEqual {
+				kp.ObjectMeta.ResourceVersion = existingKp.ObjectMeta.ResourceVersion
+				if err = k8sClient.Update(ctx, &kp); err != nil {
+					logger.Error(err, "failed to configure existing KyvernoPolicy", "KyvernoPolicy.Name", existingKp.Name, "KyvernoPolicy.Namespace", existingKp.Namespace)
+					return
+				}
+				logger.Info("KyvernoPolicy configured", "KyvernoPolicy.Name", existingKp.Name, "KyvernoPolicy.Namespace", existingKp.Namespace, "Reason", reason)
 			}
```

Comment:
If the objects are equal, a "continue" statement is needed so as to avoid calling the adapteutil.UpdateNpStatus